ev3dev/core.py

```diff
@@ -1936,6 +1936,7 @@ class Sensor(Device):
     '_decimals',
     '_driver_name',
     '_mode',
+    '_mode_value',
```

Comment:
The PR title says 'Infrared sensor', but this looks like a move to introduce caching for the mode value. Should this be split into a couple of PRs?

Reply:
Will pull this part out into a new PR